src/jrd/DbCreators.h

```diff
@@ -36,7 +36,14 @@
 
 namespace Jrd {
 
-bool checkCreateDatabaseGrant(const Firebird::MetaString& userName, const Firebird::MetaString& trustedRole,
+enum class CreateGrant
+{
+	none,		// This user cannot create databases
```

Comment:
Our `enum class` members uses ALL_UPPER_CASE convention.

Reply:
Isn't it only for old code which was converted from macros? All-uppercase identifiers are ugly.